cdhweb/pages/migration_utils.py

```diff
@@ -0,0 +1,154 @@
+import json
+import logging
+from datetime import datetime
+
+from django.core.serializers import serialize
+from django.db import connection
+from django.db.migrations.executor import MigrationExecutor
+from django.test import TestCase
+
+logger = logging.getLogger('wagtail.core')
+
+# see for an explanation of django-treebeard & the `path` attribute:
+# http://www.agilosoftware.com/blog/django-treebard-and-wagtail-page-creation/
+def get_parent(apps, page):
+    '''Find the parent of a wagtail page instance. Always returns the underlying
+    :class:`wagtail.core.models.Page` rather than a subclass.
+    '''
+
+    # if the path is 4 digits, we're at the root, so there is no parent
+    if len(page.path) == 4:
+        return None
+
+    # otherwise remove the final 4 digits to get the parent's id (pk)
+    Page = apps.get_model('wagtailcore', 'Page')
+    return Page.objects.get(path=page.path[:-4])
+
+# add_child utility method adapted from:
+# http://www.agilosoftware.com/blog/django-treebard-and-wagtail-page-creation/
+# see also:
+# https://github.com/wagtail/wagtail/blob/stable/2.3.x/wagtail/core/models.py#L442
+def add_child(apps, parent_page, klass, **kwargs):
```

Comment:
Same question about this one as `get_parent`. The docs mention a method, but it doesn't seem to be documented and I don't know if we need a particular version of wagtail before it's available.